Documentation/Security/GeneralInformation/Index.rst

```diff
@@ -56,17 +56,23 @@ backend may be changed and appropriate training for editors may be
 required. The content rendering may change, so that updates in
 `TypoScript`, templates or `CSS` code may be necessary. With LTS and
 Sprint Releases also the system requirements (for example PHP or MySQL
-version) may change. For a patch level releases (i.e.
-changing from release `9.5.0` to `9.5.1`) the database structure and
+version) may change. For a patch level release (i.e.
+changing from release `11.5.0` to `11.5.1`) the database structure and
 backend will usually not change and an update will only require the
 new version of the source code.
 
 List of TYPO3 LTS releases:
 
-* v7 (7.6.0 LTS): versions 7.0 through 7.5 do not receive security upgrades any longer
-* v8 (8.7.0 LTS): versions 8.0 through 8.6 do not receive security upgrades any longer
-* v9 (9.5.0 LTS): versions 9.0 through 9.4 do not receive security upgrades any longer
-* v10 (10.4.0 LTS): versions 10.0 through 10.3 do not receive security upgrades any longer
+*   v7 (7.6.0 eLTS): No free security updated. Extended Long Term support can
```

Comment:
As 7.6 falls out of eLTS support end of November this year, we can remove it: https://typo3.com/services/extended-support-elts